bridges/centralized-ethereum/src/actors/dr_database.rs

```diff
@@ -70,11 +101,22 @@ impl Message for GetLastDrId {
 impl Handler<SetDrInfoBridge> for DrDatabase {
     type Result = ();
 
-    fn handle(&mut self, msg: SetDrInfoBridge, _ctx: &mut Self::Context) -> Self::Result {
+    fn handle(&mut self, msg: SetDrInfoBridge, ctx: &mut Self::Context) -> Self::Result {
         let SetDrInfoBridge(dr_id, dr_info) = msg;
         self.dr.insert(dr_id, dr_info);
 
         self.max_dr_id = cmp::max(self.max_dr_id, dr_id);
+
+        // Persist Data Request Database
+        let bridge_db = self.clone();
```

Comment:
Maybe this clone can be avoided by doing something like this: (not tested)

```rust
let f = storage_mngr::put(&BRIDGE_DB_KEY, &self);
let fut = async move {
    let res = f.await;
}
```

Reply:
![image](https://user-images.githubusercontent.com/44392885/115431458-9e577500-a205-11eb-83e7-be6231a60f80.png)

Reply:
Interesting, it works if you remove the `&` from `&self`:

    let f = storage_mngr::put(&BRIDGE_DB_KEY, self);